recipe/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "python-ffmpeg" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/jonghwanhyeon/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: a7977cc3d8658f94a33cd15c5710e008c9e5442df32f57e7df8cbb4ad38a97f3
+
+build:
+  # ffmpeg is not available for s390x
+  skip: true  # [s390x]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - python
+    - pyee
+    - typing_extensions
+    - ffmpeg
+
+test:
+  requires:
+    - pip
+    - pytest
```

Comment:
Could you add `pytest-asyncio` or `anyio` to run all skipped tests (just to be sure that all works fine)? In the logs, I see the next message:
```
  You need to install a suitable plugin for your async framework, for example:
    - anyio
    - pytest-asyncio
    - pytest-tornasync
    - pytest-trio
    - pytest-twisted
```

I think that's why 4 tests skipped:
`=================== 4 passed, 4 skipped, 8 warnings in 0.66s ===================`

Reply:
ah that's cool! Added, thanks!